src/imageUtil.ml

```diff
@@ -239,6 +261,21 @@ let int32_of_str4 s =
   ((Int32.of_int (int_of_char s.[2])) << 8) ++
   (Int32.of_int (int_of_char s.[3]))
 
+(*
+ * Converts a string of size 4 into an Int32 using little endian
+ * Arguments:
+ *   - s : the string (should have size 4 at least).
+ * Returns an Int32.
+ *)
+let int32_of_str4_le s =
```

Comment:
You can avoid having the `Int32` all over if you bind some local functions:
```ocaml
let int32_of_str4_le s =
  let (<<), (++) = Int32.(shift_left, add) in
  let of_int_at x = Int32.of_int (int_of_char s.[x]) in
   (of_int_at 0) ++
   (of_int_at 1 << 8) ++
   (of_int_at 2 << 16) ++
   (of_int_at 3 << 24)
```

alternatively you can open the `Int32` module locally:
```ocaml
let int32_of_str4_le s =
  let open Int32 in
  let (<<) = shift_left in
  let (++) = add in
  ((of_int (int_of_char s.[0])) ) ++
  ((of_int (int_of_char s.[1])) << 8) ++
  ((of_int (int_of_char s.[2])) << 16) ++
  (of_int (int_of_char s.[3]) << 24)
```